waku/v2/protocol/waku_noise/noise_utils.nim

```diff
@@ -406,4 +406,9 @@ proc deserializePayloadV2*(payload: seq[byte]): Result[PayloadV2, cstring]
   payload2.transportMessage = payload[i..i+transportMessageLen-1]
   i += transportMessageLen
 
-  return ok(payload2)
\ No newline at end of file
+  return ok(payload2)
+
+# Simple utility that checks if the given variable is "default", 
+# Therefore, it has not been initialized
+proc isDefault*[T](defaultyVar: T): bool = 
+  result = defaultyVar == default(T)
```

Comment:
We do not use `result`. Please, check the following Status' nim style guide entry: https://status-im.github.io/nim-style-guide/language.result.html

Reply:
Nim's expression based returns should allow you to do something like

```suggestion
  defaultyVar == default(T)
```

Is it meant to be `defaultyVar` rather than `defaultVar`? :)

Reply:
After seeing @s1fr0 's suggestion, replacing `defaultyVar` with `value`

Reply:
Resolved in [dc0bc0b](https://github.com/status-im/nwaku/pull/1104/commits/dc0bc0b83102f3d224f8f1b4e6abd64852f56ec2)